Add Audio Over IP

Mo-way opened the pull request:
**https://github.com/Mo-way/awesome-aoip**

[Audio Over IP](https://www.avid.com/resource-center/audio-over-ip-avb-and-dante-what-todays-music-producer-should-know) - Reliably transmit uncompressed, low-latency audio over an IP (layer 3) network.

Many people coming from Audio engineering aren't necessarily experienced with networking, 
so this list should help them (and others) get started in the field.

[PRs reviewed](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+reviewed-by%3Amo-way):
- https://github.com/sindresorhus/awesome/pull/3308
- https://github.com/sindresorhus/awesome/pull/3299
- https://github.com/sindresorhus/awesome/pull/3293
- https://github.com/sindresorhus/awesome/pull/3048


### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.** TODO

## Requirements for your pull request

- [x] Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- [x] **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive. 
- [x] **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions. Comments pointing out lint violation are allowed, but does **not** count as a review.
- [x] You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- [x] This pull request has a title in the format `Add Name of List`. It should not contain the word `Awesome`. 
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- [x] Your entry here should include a short description of the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb. It should not contain the name of the list.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- [x] Your entry should be added at the bottom of the appropriate category.
- [x] The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- [x] No blockchain-related lists.
- [x] The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [x] Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- [x] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- TODO: check good example for my intro, e.g. no cureted list?
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- [x] It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- [x] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- [x] The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- [x] Non-generated Markdown file in a GitHub repo.
- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [x] Not a duplicate. Please search for existing submissions.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to a maximum of half the width of the original image.
	- Don't include both a title saying `Awesome X` and a logo with `Awesome X`. You can put the header image in a `#` (Markdown header) or `<h1>`.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- [x] Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- [x] Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. The casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- [x] All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- [x] Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] Does not use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [x] Does not include a CI (e.g. GitHub Actions) badge.<br>You can still use a CI for linting, but the badge has no value in the readme.
- [x] Does not include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

Mo-way commented:
unicorn

nicholasaleks commented:
I've checked the checklist and don't see anything obvious that is missing.

I'm not an expert in Audio Over IP, but the list seems a bit short. I did some light research and perhaps you can consider also including sections on Virtual Soundcards, Monitoring & Diagnostics Tools,  Live/Archived Stream Tools and/or Developer SDKs might help expand on the content?

Also just a few minor NITs
- Typos: "plattforms", "ressources", "contianing" across the README
- A short glossary or link to terms like AES67, NMOS, PTP, and Ravenna would aid new readers
- Several tools (e.g., PAM, JackTrip, LinuxPTP) could use a more explicit AoIP relevance line

Mo-way commented:
Hi @nicholasaleks, thanks for the feedback!

- Fixed typos, forgot to spell check final version, sorry!
- I'd love to add awesome Virtual soundcards. The only one I personally can truly recommend is the open source kernel module I added on the list. For Dante it's trivial to find the virtual soundcard as you did, because Audinate is the only vendor.  For Ravenna I'm unsure: adding the ones I know are working vs. waiting for people to add the ones they prefer.
- For other sections it's a similar decision. Just add something that looks nice and seems to be fine vs. waiting for people who actually work with it a lot and can tell if it deserves the spot.

The list is short, because I pretty harshly went for quality instead of quantity. 

I'm all for making it easier. Why did the links to AES67, Ravenna and Dante on the top not suffice? 
I never saw a Glossary section in an awesome list, but adding one here makes sense, your're right.

hstsethi commented:
The "Footnotes" heading is missing in the TOC. 

I have sent a PR for it.

Mo-way commented:
@sindresorhus Getting no more feedback from others. Could you review please?

twanamhmd430-lang reviewed (changes requested):
Reply- ****![Uploading 1756001339694.jpg …]()

<!-- Error while uploading 1756038496869.jpg --><!-- Error while uploading IMG_۲۰۲۵۰۸۲۴_۱۱۳۱۴۷.jpg --><!-- Error while uploading 175601609<!-- Error while uploading 1756016090459.jpg -->9.jpg …]()

<!-- Error while uploading tp<!-- Error while uploading 175601606<!-- Error while uploading tp_share_image_۲۰۲۵۰۸۲۴_۰۵۰۸۴۰.jpg --><!-- Error while uploadi<!-- Error while uploading 175600134<!-- Error while uploading Screenshot_۲۰۲۵۰۸۲۴-۰۴<!-- Error while uploading Ir0340766954003_and_36qReB4yt<!-- Error while uploading 1755993322932.jpg -->t_۲۰۲۵۰۸۲۴-۰۴۴۹۰۹.png …]()

![Uploading Ir0340766954003_and_36qReB4ytUE8b.jpg …]()

![Uploading 1755993322932.jpg …]()

![Uploading Ir0340766954003_and_36qReB4ytUE8b.jpg …]()

<!-- Error while uploading Ir0340766954003_and_36qReB4yt<!-- Error while uploading 1755989763695.jpg --><!-- Error while uploadi<!-- Error while uploading Screenshot_۲۰۲۵۰۸۲۳-۰۰۱۹۳۳.png -->creenshot_۲۰<!-- Error while uploading Sepino-2025-08-22T20_23_49.jpg -->8-22T20_23_49.jpg …]()

![Uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8<!-- Error while uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8nj9wsk4cp (1).jpg -->nj9wsk4cp (1).jpg …]()

<!-- Error while uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8nj9wsk4cp.jpg --><!-- Error while uploading ETH_0x4D5DFf23b79b3A07AeBa1E24551dF29611A21Bf5.jpg --><!-- Erro<!-- Error while uploading ETH_0x4D5DFf23b79b3A07AeBa1E24551dF29611A21Bf5 (1).jpg -->3b79b3A07AeB<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۴۴۱۱.png -->ng IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۴۴۱۱.png …]()

![Uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۶۷۸۹.png -->ading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۰۱۵۶.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۶۷۸<!-- Error while uploading IMG_۲۰<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۲۹۹۶.png -->ng IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۲۹۹<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳<!-- Error while uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۲۱۷۱۸.png -->_۲۰۲۵۰۸۲۱_۲۳۲۳۲۳۵۲۶.png <!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱۵۰۹۵.png -->_۲۰۲۵۰۸۲۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱۰۰۰۹.png --><!-- Error while uploading IMG_۲۰<!-- Error while uploading IMG_۲۰<!-- Error while uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۲۸۵۵.png --><!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۰۶۵۵.png -->۲۵۰۸۲۱_۲۳۲۳۱۰۰۰۹.png <!-- Erro<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۳۲۱۲.png -->-->_۲۰۲۵۰۸۲۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۴۹۷۹۲.png -->_۲۳۲۳۰۴۲۸۴.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۲۸۵۵.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۰۶۵۵.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۷۹۲۴.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۶۲۱۴.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۳۲۱۲.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۱۵۳۹.png …]()

![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۴۹۷۹۲.png …]()

****

twanamhmd430-lang commented:
Repley> - ****![Uploading 1756001339694.jpg …]()
> 
> <!-- Error while uploading 1756038496869.jpg --><!-- Error while uploading IMG_۲۰۲۵۰۸۲۴_۱۱۳۱۴۷.jpg --><!-- Error while uploading 175601609<!-- Error while uploading 1756016090459.jpg -->9.jpg …]()
> 
> <!-- Error while uploading tp<!-- Error while uploading 175601606<!-- Error while uploading tp_share_image_۲۰۲۵۰۸۲۴_۰۵۰۸۴۰.jpg --><!-- Error while uploadi<!-- Error while uploading 175600134<!-- Error while uploading Screenshot_۲۰۲۵۰۸۲۴-۰۴<!-- Error while uploading Ir0340766954003_and_36qReB4yt<!-- Error while uploading 1755993322932.jpg -->t_۲۰۲۵۰۸۲۴-۰۴۴۹۰۹.png …]()
> 
> ![Uploading Ir0340766954003_and_36qReB4ytUE8b.jpg …]()
> 
> ![Uploading 1755993322932.jpg …]()
> 
> ![Uploading Ir0340766954003_and_36qReB4ytUE8b.jpg …]()
> 
> <!-- Error while uploading Ir0340766954003_and_36qReB4yt<!-- Error while uploading 1755989763695.jpg --><!-- Error while uploadi<!-- Error while uploading Screenshot_۲۰۲۵۰۸۲۳-۰۰۱۹۳۳.png -->creenshot_۲۰<!-- Error while uploading Sepino-2025-08-22T20_23_49.jpg -->8-22T20_23_49.jpg …]()
> 
> ![Uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8<!-- Error while uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8nj9wsk4cp (1).jpg -->nj9wsk4cp (1).jpg …]()
> 
> <!-- Error while uploading BTC_bc1qnsmkvuqn4mqag5q6nvuu7u5hncrj8nj9wsk4cp.jpg --><!-- Error while uploading ETH_0x4D5DFf23b79b3A07AeBa1E24551dF29611A21Bf5.jpg --><!-- Erro<!-- Error while uploading ETH_0x4D5DFf23b79b3A07AeBa1E24551dF29611A21Bf5 (1).jpg -->3b79b3A07AeB<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۴۴۱۱.png -->ng IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۴۴۱۱.png …]()
> 
> ![Uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۶۷۸۹.png -->ading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۴۰۱۵۶.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۶۷۸<!-- Error while uploading IMG_۲۰<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۲۹۹۶.png -->ng IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳۲۹۹<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۳<!-- Error while uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۲۱۷۱۸.png -->_۲۰۲۵۰۸۲۱_۲۳۲۳۲۳۵۲۶.png <!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱۵۰۹۵.png -->_۲۰۲۵۰۸۲۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۱۰۰۰۹.png --><!-- Error while uploading IMG_۲۰<!-- Error while uploading IMG_۲۰<!-- Error while uplo<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۲۸۵۵.png --><!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۰۶۵۵.png -->۲۵۰۸۲۱_۲۳۲۳۱۰۰۰۹.png <!-- Erro<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۳۲۱۲.png -->-->_۲۰۲۵۰۸۲۱<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵<!-- Error while uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۴۹۷۹۲.png -->_۲۳۲۳۰۴۲۸۴.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۲۸۵۵.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۳۰۰۶۵۵.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۷۹۲۴.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۶۲۱۴.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۳۲۱۲.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۵۱۵۳۹.png …]()
> 
> ![Uploading IMG_۲۰۲۵۰۸۲۱_۲۳۲۲۴۹۷۹۲.png …]()
> 
> ****

twanamhmd430-lang commented:
🤔
